packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -27,11 +27,12 @@ class NodeCreate(InputSchemaWithoutCamelCase):
 
 class NodePatch(InputSchemaWithoutCamelCase):
     service_version: ServiceVersion = FieldNotRequired(alias="version")
-    label: str = Field(None)
-    inputs: InputsDict = Field(None)
+    label: str = FieldNotRequired()
+    inputs: InputsDict = FieldNotRequired()
+    inputs_required: list[InputID] = FieldNotRequired(alias="inputsRequired")
```

Comment:
this reads better
```suggestion
    required_inputs: list[InputID] = FieldNotRequired(alias="inputsRequired")
```

Reply:
I would like to keep it consistent with other `inputs_*` naming we use